.github/workflows/dotnet.yml

```diff
@@ -13,10 +13,25 @@ jobs:
     - uses: actions/setup-dotnet@v2
       with:
         dotnet-version: '5.0.x'
-    - name: Restore dependencies
-      run: dotnet restore
-    - name: Run .NET 5.0 tests
-      run: dotnet test SharpXMPP.NUnit --no-restore --configuration Release -f net5.0
+    - name: Add msbuild to PATH
+      if: matrix.os == 'windows-latest'
+      uses: microsoft/setup-msbuild@v1.1
+    - name: Build console app
+      if: matrix.os == 'ubuntu-latest'
```

Comment:
We can build this on any OS, actually?

Reply:
Addressed in https://github.com/vitalyster/SharpXMPP/pull/151